expressions.html

```diff
@@ -587,11 +579,17 @@
 			<td></td>
 		</tr>
 		<tr title="Arrays with custom syntax">
-			<td mv-expressions="{{ }}">{{ [1, 2, 3, 4] }}</td>
+			<td mv-expressions="{{ }}">
+				<span mv-expressions="{{ }}">
+					{{ [1, 2, 3, 4] }}
+				</span>
+			</td>
 			<td>[1, 2, 3, 4]</td>
 		</tr>
 		<tr>
-			<td mv-expressions="{{ }}">{{ [-1, 2, -3] }}</td>
+			<td mv-expressions="{{ }}">
+				<span mv-expressions="{{ }}">{{ [-1, 2, -3] }}</span>
```

Comment:
You don't need this here, `mv-expressions` inherits. Also, you don't need a span at all, actually.
Same applies to the other one.